data/brands/shop/second_hand.json

```diff
@@ -9,6 +9,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "Salvos",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "Salvos",
+        "brand:wikidata": "Q120646407",
+        "name": "Salvos",
+        "shop": "second_hand"
```

Comment:
Looks like it is operated by the salvation army. So I think I agree with you it should be moved to charity. I just read thrift store and think second hand. But I found somewhere that says thrift & charity stores. So, I think it could be in either, but more geared towards charity.

Reply:
Sounds good to me. It might be a good idea to verify that [existing locations](https://overpass-turbo.eu/s/1Et7) have been tagged correctly.